server/channels/app/export.go

```diff
@@ -491,6 +504,61 @@ func (a *App) exportAllUsers(ctx request.CTX, job *model.Job, writer io.Writer,
 	return profilePictures, nil
 }
 
+func (a *App) exportAllBots(ctx request.CTX, job *model.Job, writer io.Writer, includeProfilePictures bool) ([]string, *model.AppError) {
+	afterId := strings.Repeat("0", 26)
```

Comment:
This is a common mistake used in a lot of places. We can just use "" here and it should work. Probably we can do a clean sweep of all these later.